rdrf/rdrf/admin.py

```diff
@@ -492,18 +491,18 @@ def registry(obj):
 
 admin.site.register(ContextFormGroup, ContextFormGroupAdmin)
 
-
-if has_feature('adjudication'):
+def register_adjudication_admin():
     admin.site.register(Notification, NotificationAdmin)
     admin.site.register(Adjudication, AdjudicationAdmin)
     admin.site.register(AdjudicationDefinition, AdjudicationDefinitionAdmin)
     admin.site.register(AdjudicationRequest, AdjudicationRequestAdmin)
     admin.site.register(AdjudicationResponse, AdjudicationResponseAdmin)
     admin.site.register(AdjudicationDecision, AdjudicationDecisionAdmin)
 
+register_adjudication_admin()
```

Comment:
There are no global features settings any more. I removed the function ae11152f9e4bb4cfc2097f3f08652bca1ea31438.